tools/minimerge/import.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+DIR="$(realpath "$(dirname $0)")"
+REPO_DIR="$(realpath "${DIR}/../../")"
+TOOL="${DIR}/minimerge_tool"
+METADATA_FILE="${DIR}/cuttlefish_import_metadata.txt"
+CUTTLEFISH_REPO_URL="https://android.googlesource.com/device/google/cuttlefish"
+
+echo "Importing AOSP's device/google/cuttlefish project"
+
+# Do these early to avoid doing unnecessary work when these fail
+# Find last imported commit
+stat "${METADATA_FILE}" >/dev/null
+last_imported_commit="$(grep '^[[:space:]]*last_import[[:space:]]*:' "${METADATA_FILE}" | cut -d: -f2 | grep -o -E '[^[:space:]]+' )"
+# Find current commit
+pushd "${DIR}" >/dev/null
+[[ `git status --porcelain` ]] >/dev/null && (echo "Repository is not in clean state"; exit)
```

Comment:
Is `exit` here intended to stop the script? It seems to me like it exits the subshell created by the parentheses.

For example with this `test.sh`:
```bash
#!/bin/bash
(echo "hello"; exit; echo "abc")
(echo "goodbye"; exit; echo "def")
```
Running it produces:
```shell
$ bash test.sh
hello
goodbye
```

Reply:
Fixed it